documentation/BookAnthology.json

```diff
@@ -1,6 +1,6 @@
 {
   "type" : "Publication",
-  "identifier" : "5a0d2a7d-d0f3-4d3c-918f-067d6a37eef7",
+  "identifier" : "955fab17-542d-44fa-a6d0-a2100d8b9ef3",
```

Comment:
Do we need id as part of the ontology for Publication or is that implicit?

Reply:
In essence, the nature of the identifier is not important, however, we have platform expectations, so it becomes important. I will create a code example as a suggested fix for this.